pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -250,6 +251,44 @@ func (s *SSOSettingsService) encryptSecrets(ctx context.Context, settings map[st
 	return result, nil
 }
 
+func (s *SSOSettingsService) Run(ctx context.Context) error {
+	ticker := time.NewTicker(1 * time.Minute)
```

Comment:
Should we make this time configurable?

Reply:
Configurable from the ini file or just make it a constant? I'm wondering if we can put this in the ini file and let the users opt out from running this background reload process.

Reply:
Yes putting this in ini file. Great point that it would be possible to opt out from there